tests/lib/ecma-version.js

```diff
@@ -141,6 +141,30 @@ describe("ecmaVersion", () => {
             }, /Invalid ecmaVersion/);
         });
 
+        it("Should throw error using invalid year", () => {
```

Comment:
This test case has the same name as the previous test case. Is that intentional?

One possible suggestion (feel free to disregard):

```suggestion
        it("Should throw error when non-numeric year is provided", () => {
```